src/protos/stub.proto

```diff
@@ -0,0 +1,174 @@
+package bicyclade;
+
+message Container {
+  optional PlayerAction player_action;
+  optional ServerAction server_action;
+  optional SpectatorAction spectator_action;
+}
+
+message Game {
+  optional uint32 id;
+  repeated Player players;
+  repeated Card cards;
+  repeated Offering offerings;
+  repeated GodType gods;
```

Comment:
a god has a position in the godlist no ? not only godType

Reply:
Agreed